modules/creating-a-github-application-in-administrator-perspective.adoc

```diff
@@ -0,0 +1,25 @@
+:_content-type: PROCEDURE
+[id="creating-a-github-application-in-administrator-perspective_{context}"]
+
+= Creating a GitHub application in administrator perspective
+
+As a cluster administrator, you can configure your GitHub application with the OpenShift Container Platform cluster to use pipeline as code. This configuration allows you to execute a set of tasks required for build deployment.
+
+.Prerequisites
+You have installed the Red Hat OpenShift pipelines 1.8 operator from the custom catalog source.
+
+.Procedure
+. In the administrator perspective, navigate to *Pipelines* using the navigation pane.
+. Click *Setup GitHub App* on the *Pipelines* page.
+. Enter your GitHub application name, for example, pipelines-ci-clustername-ss-test.
+. Click the *Setup* button.
+. Enter your Git password when prompted in the browser.
+. Click *Create GitHub App for `username`*. The variable `username` denotes your actual user name.
+
+.Verification
+When the Git application is created, you are navigated back to the OpenShift console that notifies successful setup of the GitHub application and provides all the details of the application.
+
+image::Github-app-details.png[]
+
+The details of the created application are saved as a secret in the `openShift-pipelines` namespace.
+To view the details of the GitHub applications created, navigate to *Pipelines* and click the *View GitHub App* button. The details include the name, link, and secret associated with the application.
```

Comment:
```suggestion
To view the details of the GitHub applications created, navigate to *Pipelines* and click *View GitHub App*. The details include the name, link, and secret associated with the application.
```

Per the IBM entry on "Button": `In general, refer to a button by its label alone.`